repo2docker/app.py

```diff
@@ -549,6 +557,24 @@ def start_container(self):
 
         container = client.containers.run(self.output_image_spec, **run_kwargs)
 
+        # copies the new contents of the image into container
+        if self.reuse_image:
+            image_workdir = container.image.attrs["Config"]["WorkingDir"]
+
+            # local path to repo clone
+            repo_path = os.path.abspath(self.git_workdir) + "/"
+            repo_archive = archive_repo(repo_path)
+
+            # delete old container contents
+            cmd = "sh -c 'rm -rf {}/*'".format(image_workdir)
+            container.exec_run(cmd, stderr=True, stdout=True)
+
+            copied = container.put_archive(image_workdir, repo_archive)
+            shutil.rmtree(self.git_workdir, ignore_errors=True)
+            if not copied:
+                self.log.error("Failed to copy repo contents into containe")
```

Comment:
Just a little typo here, should be "container".

Reply:
Fixed! Thanks :)